testhelpers/mining.go

```diff
@@ -33,6 +50,35 @@ func (t *TestWorkerPorcelainAPI) MinerGetWorkerAddress(_ context.Context, _ addr
 	return t.workerAddr, nil
 }
 
+// Queryer returns a queryer object for the given tipset
```

Comment:
This package imports consensus so why not resuse the test Queryer defined in `consensus/testing.go` which looks very similar?

Reply:
👍